README.md

```diff
@@ -38,4 +38,11 @@ Green Light Health App helps businesses prevent spread of illness in their locat
 # Product Backlog:
 Find the [Product Backlog](https://github.com/jorgecotillo/green_light_health/projects/1)
 under github projects.
+
 # Definition of Ready:
+Our D.O.R. requires that the team be able to determine "what needs to be done" and the "amount of work" required to complete a given story.
+The below checklists should be complete to ensure a story is Ready.
+- [ ] Does the story capture and describe a reasonable and unit of work?
+- [ ] Has the amount of work been estimated using an approved measurement?
```

Comment:
Not sure what the rest of the team members think, but my preference would be to change the bullet points from questions to statements:

e.g. -> [ ] Work has been estimated using approved measurement (t-shirt size)

Reply:
I'm open to that format, sure